data/path/path.go

```diff
@@ -23,6 +24,16 @@ func GetValue(value interface{}, path string) (interface{}, error) {
 	var err error
 	var newPath string
 
+	//To interface if it is an string
```

Comment:
@lixingwang can you explain what this is for?

Reply:
@fm-tibco Sometimes, the value in the field is a json string,  Convert to go object to get the value from json.